deploy/helm/kubecf/templates/brain-tests.yaml

```diff
@@ -0,0 +1,223 @@
+# psp-role       | predefined ? -> scf default account
+# configgin-role | predefined ? -> scf default account
+
+# Service account, roles, and cluster role for the brain tests.
+# Raw kube objects going into the chart.
+# TODO: Make this conditional.
+
+# Service account "tests-brain" is used by the instance group "acceptance-tests-brain"
+---
+apiVersion: "v1"
+kind: "ServiceAccount"
+metadata:
+  name: "{{ .Values.deployment_name }}-tests-brain"
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app.kubernetes.io/component: "{{ .Values.deployment_name }}-tests-brain"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
+    app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ include "kubecf.chart" . }}
+
+# Role "test-role-brain" only used by account "tests-brain"
+# The permissions are a combination of the SCF roles
+# configgin-role, psp-role, and test-brain-role.
```

Comment:
We shouldn't need to refer back to SCF when looking at this anymore. (It makes sense as a comment in the PR, not as a thing to commit on merge.)

Reply:
Just in the PR it is easy to forget.

Reply:
Done with commit b4a6f34cf652a6e983167b8319a88217acada94f